pysb/kappa.py

```diff
@@ -30,6 +29,99 @@
 except ImportError:
     pass
 
+# Cached value of Kappa program paths
+_kasim_path = None
+_kasa_path = None
+
+def _check_dist_dir(kappa_prog, dist_dir):
+    kappa_prog_path = os.path.join(dist_dir, kappa_prog)
+    if os.access(kappa_prog_path, os.F_OK):
+        return kappa_prog_path
+    else:
+        return None
+
+def set_kappa_path(dir):
```

Comment:
Don't shadow `dir`. I know the original code from bng.py does it, but that should be fixed too!